Engines/Wine/Verbs/d3drm/script.js

```diff
@@ -0,0 +1,34 @@
+include(["engines", "wine", "engine", "object"]);
+include(["engines", "wine", "plugins", "override_dll"]);
+include(["utils", "functions", "net", "resource"]);
+include(["engines", "wine", "verbs", "luna"]);
+include(["utils", "functions", "filesystem", "files"]);
+include(["utils", "functions", "filesystem", "extract"]);
+
+/**
+* Verb to install d3drm
+* @returns {Wine} Wine object
+*/
+Wine.prototype.d3drm = function () {
+    var setupFile = new Resource()
+        .wizard(this.wizard())
+        .url("https://download.microsoft.com/download/E/E/1/EE17FF74-6C45-4575-9CF4-7FC2597ACD18/directx_feb2010_redist.exe")
+        .checksum("a97c820915dc20929e84b49646ec275760012a42")
+        .name("directx_feb2010_redist.exe")
+        .get();
+    this.wizard().wait(tr("Please wait while {0} is installed ...", "d3drm"));
+    new CabExtract()
+        .archive(setupFile)
+        .to(this.prefixDirectory() + "/drive_c/d3drm/")
+        .extract(["-L", "-F", "dxnt.cab"]);
+    new CabExtract()
+        .archive(this.prefixDirectory() + "/drive_c/devenum/dxnt.cab")
```

Comment:
This looks strange to me, where comes the `/drive_c/devenum/dxnt.cab` file from?

Reply:
From the previous `CabExtract`. The verb have to extract `dxnt.cab` from the .exe file it downloads. Then it extract the desired .dll file from that .cab archive.

Reply:
Yes but take a look at the folder name.
As far as I understand your code you first extract the `dxnt.cab` file to `/drive_c/d3drm/dxnt.cab`.
Then you try to extract the `d3drm.dll` from `/drive_c/devenum/dxnt.cab`.
The folder names are different.

Reply:
You are correct.